openpgp/internal/ecc/curves.go

```diff
@@ -13,20 +13,20 @@ type Curve interface {
 
 type ECDSACurve interface {
 	Curve
-	MarshalPoint(x, y *big.Int) []byte
-	UnmarshalPoint([]byte) (x, y *big.Int)
+	MarshalIntegerPoint(x, y *big.Int) []byte
+	UnmarshalIntegerPoint([]byte) (x, y *big.Int)
 	MarshalIntegerSecret(d *big.Int) []byte
 	UnmarshalIntegerSecret(d []byte) *big.Int
 	GenerateECDSA(rand io.Reader) (x, y, secret *big.Int, err error)
 	Sign(rand io.Reader, x, y, d *big.Int, hash []byte) (r, s *big.Int, err error)
 	Verify(x, y *big.Int, hash []byte, r, s *big.Int) bool
-	Validate(x, y *big.Int, secret []byte) error
+	ValidateECDSA(x, y *big.Int, secret []byte) error
```

Comment:
Since you changed this one, should the one for `EdDSACurve` be `ValidateEdDSA`?

Reply:
It's not required (as there are no 2 curves that are both ECDSA & EdDSA or ECDH & EdDSA) but for symmetry it makes perfect sense. Will do